### DISCUSSION TITLE
can't see github copilot chat in vscode

### DISCUSSION THREAD

**Discussion by AssafSG** [community/community#112327] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have installed the plugin (copilot chat), and the other plugin of copilot. 
I can see the icon on the bottom right of the screen.
Click on the icon -> the status menu opens above (on the command menu)
Click on Github copilot chat -> nothing happens.

Sometimes I see it on the right, but when I click it doesn't open, and the icon disappears

**Comment by digitarald** [comment]
Please follow this guide: https://github.com/microsoft/vscode-copilot-release/issues/436
